app/src/org/commcare/services/CommCareSessionService.java

```diff
@@ -197,14 +199,20 @@ public IBinder onBind(Intent intent) {
     /**
      * Show a notification while this service is running.
      */
+    @SuppressLint("UnspecifiedImmutableFlag")
     public void showLoggedInNotification(@Nullable User user) {
         //We always want this click to simply bring the live stack back to the top
         Intent callable = new Intent(this, DispatchActivity.class);
         callable.setAction("android.intent.action.MAIN");
         callable.addCategory("android.intent.category.LAUNCHER");
 
         // The PendingIntent to launch our activity if the user selects this notification
-        PendingIntent contentIntent = PendingIntent.getActivity(this, 0, callable, 0);
+        // The FLAG_IMMUTABLE flag was added in API level 23

```

Comment:
nit: don't think these comments are necessary as it seems evident from the sdk version check.